server/tests/tables_structs/status_mapping_struct.js

```diff
@@ -0,0 +1,8 @@
+const StatusMappingStruct = function(table) {
```

Comment:
Where is the mapping saved? Should it be a part of the migration?

Reply:
Yep, it _is_ part of the migration, that part was already merged last month: 
https://github.com/meirim-org/meirim/blob/master/server/migrations/20221031144432_add_status_mapping.js